civic_scraper/platforms/legistar/scraper.py

```diff
@@ -0,0 +1,82 @@
+import civic_scraper
+from civic_scraper import base
+from civic_scraper.base.asset import Asset, AssetCollection
+from civic_scraper.base.cache import Cache
+
+from legistar.events import LegistarEventsScraper
+
+from datetime import datetime
+from pathlib import Path
+from urllib.parse import urlparse, parse_qs
+
+# Scrape today's agendas and minutes from a Legistar site
+class LegistarSite(base.Site):
+
+    def __init__(self, base_url, legistar_instance, state, cache=Cache(), parser_kls=None, timezone=None):
```

Comment:
why do we need to know the state?

Reply:
seems like we could derive the legistar_instance from the url?

Reply:
we need to know the state for the file name; I'm not sure how likely it is that two legistar instances exist for municipalities withe same name but in different states, for example, Canton OH and Canton GA. Is there a better way of accounting for this possibility?

And yes, we can get the legistar_instance from the url. I'll work through that.

Reply:
there is not way for a legistar url to be the same for two different municipalities. 

somebody definitely needs to know the relationship between scrapers and munipalities, but why doe *this* scraper have to know that relationship.

Reply:
ah, right, using the url will already involve unique municipality identifiers because the url uses unique municipality identifiers

about the second question/comment, it seems to me this scraper should know which municipality it's working with since it's outputting the asset files, which we have been naming in part based on the municipality. Unless we decide to name the asset files differently. file names seem incidental to the main goal of the scraper, though